deployment/deploy_capture/bundle.spec

```diff
@@ -85,67 +85,72 @@ elif platform.system() == 'Linux':
-	          console=False,
-	          resources=['pupil-capture.ico,ICON,GLFW_ICON'])
-	coll = COLLECT(exe,
-	               a.binaries,
-	               a.zipfiles,
-	               a.datas,
-	               [('glfw3.dll', '../../pupil_src/shared_modules/external/glfw3.dll','BINARY')],
-	               [('glfw3.lib', '../../pupil_src/shared_modules/external/glfw3.lib','BINARY')],
-	               [('glfw3dll.lib', '../../pupil_src/shared_modules/external/glfw3dll.lib','BINARY')],
-	               [('opencv_ffmpeg2411.dll', os.path.join(python_path, 'opencv_ffmpeg2411.dll'),'BINARY')],
-	               [('_videoInput.lib', os.path.join(python_path, '_videoInput.lib'),'BINARY')],
-	               [('msvcp110.dll', os.path.join(system_path, 'msvcp110.dll'),'BINARY')],
-	               [('msvcr110.dll', os.path.join(system_path, 'msvcr110.dll'),'BINARY')],
-	               [('msvcr120.dll', os.path.join(system_path, 'msvcr120.dll'),'BINARY')],
-	               [('OpenSans-Regular.ttf', os.path.join(package_path, 'pyglui/OpenSans-Regular.ttf'),'DATA')],
+        import sys, os, os.path
+
+        np_path = os.path.dirname(numpy.__file__)
+        np_dlls = glob.glob(np_path + '/core/*.dll')
+        np_dll_list = [] 
+
+        for dll_path in np_dlls:
+            dll_p, dll_f = ntpath.split(dll_path)
+            np_dll_list += [(dll_f, dll_path, 'BINARY')]
+        system_path = os.path.join(os.environ['windir'], 'system32')
+
+        print("Using Environment:")
+        python_path = None
+        package_path = None
+        for path in sys.path:
+                print( " -- " + path)
+                if path.endswith("scripts"):
+                        python_path = os.path.abspath(os.path.join(path, os.path.pardir))
+                elif path.endswith("site-packages"):
+                        lib_dir = os.path.abspath(os.path.join(path, os.path.pardir))
+                        python_path = os.path.abspath(os.path.join(lib_dir, os.path.pardir))
+                        package_path = path
+        if (python_path and package_path):
+                print("PYTHON PATH @ " + python_path)
+                print( "PACKAGE PATH @ " + package_path)
+        else:
+                print("could not find python_path or package_path. EXIT.")
+                quit()
+        scipy_imports = ['scipy.integrate']
+        scipy_imports += ['scipy.integrate._ode', 'scipy.integrate.quadrature', 'scipy.integrate.odepack', 'scipy.integrate._odepack', 'scipy.integrate.quadpack', 'scipy.integrate._quadpack']
+        scipy_imports += ['scipy.integrate.vode', 'scipy.integrate.lsoda', 'scipy.integrate._dop', 'scipy.special._ufuncs', 'scipy.special._ufuncs_cxx']
+
+        a = Analysis(['../../pupil_src/capture/main.py'],
+                     pathex=['../../pupil_src/shared_modules/'],
+                     binaries=None,
+                     datas=None,
+                     hiddenimports=pyglui_hidden_imports+scipy_imports+av_hidden_imports,
```

Comment:
Perhaps I am misunderstanding something here. But currently neither the bundle or src ( `python pupil_src/capture/main.py`) will run successfully if the following `.dll` are not in `pupil_src/capture`:

- glfw3.dll
- glew3.dll
- avcodec-57.dll
- avdevice-57.dll
- avfilter-6.dll
- avformat-57.dll
- avutil-55.dll
- postproc-54.dll
- swresample-2.dll
- swscale-4.dll
- opencv_world3000.dll

Are we not able to reference these files from a location where libs are installed/built on the system? 

What is the best solution here in your opinion? Adding `.dll`s to `pupil_src/shared_modules/external`?